src/v/config/configuration.cc

```diff
@@ -2392,6 +2392,23 @@ configuration::configuration()
       {.needs_restart = needs_restart::yes, .visibility = visibility::tunable},
       // TODO: make this runtime configurable.
       true)
+  , whole_cluster_restore_cluster_name(
+      *this,
+      "whole_cluster_restore_cluster_name",
+      "Specifies the cluster name to identify and use correct cluster metadata "
```

Comment:
```suggestion
      "Specifies the cluster name used to identify and retrieve the correct cluster metadata "
```
Just a suggestion for clarity, as long as it's still the correct meaning

Reply:
It is used to identify both for the upload and retrieval. I want to make this clear.

Reply:
Reworded it once again. Please let me know if you have alternative suggestions 🙇